packages/npm-mongo/wrapper.js

```diff
@@ -15,3 +45,5 @@ try {
 NpmModuleMongodbVersion = useLegacyMongo
   ? Package['npm-mongo-legacy'].NpmModuleMongodbVersion
   : Npm.require('mongodb/package.json').version;
+
+NpmModuleMongodb.connect = connect
```

Comment:
Why are we exporting or including connect here? Is it required for this to work? Is it used elsewhere?